packages/strapi-plugin-i18n/admin/src/components/CMEditViewInjectedComponents/index.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import get from 'lodash/get';
+import { useSelector } from 'react-redux';
+import { useParams } from 'react-router-dom';
+import { useContentManagerEditViewDataManager, useQueryParams } from 'strapi-helper-plugin';
+import selectI18NLocales from '../../selectors/selectI18nLocales';
+import CMEditViewLocalePicker from '../CMEditViewLocalePicker';
+
+const CMEditViewInjectedComponents = () => {
+  const { layout, modifiedData, slug } = useContentManagerEditViewDataManager();
+  const locales = useSelector(selectI18NLocales);
+  const params = useParams();
+  const id = get(params, 'id', null);
+  const currentEntityId = id === 'create' ? null : id;
+  const [{ query }] = useQueryParams();
```

Comment:
can you please put this one  3 lines up so that there's a difference between the hooks "world" and the derivation world?